target/linux/ar71xx/patches-4.14/952-qca955x-enable-ddr-wb-flush.patch

```diff
@@ -0,0 +1,49 @@
+--- a/arch/mips/ath79/common.c
++++ b/arch/mips/ath79/common.c
+@@ -49,6 +49,8 @@ void ath79_ddr_ctrl_init(void)
+ 	if (soc_is_ar913x() || soc_is_ar724x() || soc_is_ar933x()) {
+ 		ath79_ddr_wb_flush_base = ath79_ddr_base + 0x7c;
+ 		ath79_ddr_pci_win_base = 0;
++	} else if (soc_is_qca953x() || soc_is_qca955x()) {
```

Comment:
Why are you doing something that is handled by the common else case?

Reply:
drops out ath79_ddr_pci_win_base